extensions/TrackingFlags/Extension.pm

```diff
@@ -43,7 +44,19 @@ sub _tracking_flags {
 }
 
 sub _tracking_flag_names {
-  return Bugzilla::Extension::TrackingFlags::Flag->get_all_names();
+  my ($class)   = @_;
+  my $memcached = $class->memcached;
+  my $cache     = $class->request_cache;
+  my $tf_names  = $cache->{tracking_flags_names};
+
+  return @$tf_names if $tf_names;
+  $tf_names //= $memcached->get_config({key => 'tracking_flag_names'});
```

Comment:
We will need to add code to invalidate the memcache config when a tracking flag has been updated.

https://github.com/mozilla-bteam/bmo/blob/master/extensions/TrackingFlags/lib/Admin.pm#L325